starboard/common/file.cc

```diff
@@ -37,6 +38,16 @@ bool DirectoryCloseLogFailure(const char* path, SbDirectory dir) {
 
 }  // namespace
 
+void RecordFileWriteStat(int write_file_result) {
```

Comment:
I don't know what's the best practice for starboard API, is copy-paste here the only option, like no other way to reuse the code above?